shared/controlflow/codeql/controlflow/SuccessorType.qll

```diff
@@ -339,3 +339,9 @@ class RetrySuccessor extends JumpSuccessor, TRetrySuccessor {
 class JavaYieldSuccessor extends JumpSuccessor, TJavaYieldSuccessor {
   override string toString() { result = "yield" }
 }
+
+/** Holds if `t` is an abnormal exit type out of a CFG scope. */
+predicate isAbnormalExitType(SuccessorType t) {
```

Comment:
This file is reexported elsewhere, so not nice to have a top-level predicate reexported.

Reply:
How about I'll make it a member on `AbruptSuccessor` instead, then?